insecure-db-2.tf

```diff
@@ -0,0 +1,12 @@
+resource "aws_db_instance" "default2" {
+  allocated_storage    = 10
+  engine               = "mysql"
+  engine_version       = "5.7"
+  instance_class       = "db.t3.micro"
+  name                 = "mydb"
+  username             = "foo"
+  password = "foobarbaz"
+  parameter_group_name = "default.mysql5.7"
+  skip_final_snapshot  = true
+  publicly_accessible = true
+}
```

Comment:
### :warning: Aqua detected misconfiguration in your code  
**Misconfiguration ID:** AVD-AWS-0177   
**Check Name:** RDS Deletion Protection Disabled   
**Severity:** MEDIUM   
**Message:** Instance does not have Deletion Protection enabled  
**Resolution:** Modify the RDS instances to enable deletion protection.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/misconfig/n/a